source/slang/slang-ast-expr.h

```diff
@@ -677,6 +677,7 @@ class SPIRVAsmOperand
         SampledType, // __sampledType(T), this becomes a 4 vector of the component type of T
         ImageType, // __imageType(texture), returns the equivalaent OpTypeImage of a given texture typed value.
         SampledImageType, // __sampledImageType(texture), returns the equivalent OpTypeSampledImage of a given texture typed value.
+        ImagePointer, // __imagePointer(texture), returns the global variable pointer of the given texture unmangled by loads/stores
```

Comment:
I believe you can just use:
```
spirv_asm {
    OpBlah &globalVar
}
```
To reference the `globalVar` without `Load`.

Reply:
the code generated loads a pointer to the texture object if we use &, we instead need a direct call to the texture object and to skip this load, __imagePointer skips it